cmd/cluster/support/delete.go

```diff
@@ -120,15 +116,22 @@ func (o *deleteOptions) run() error {
 	}
 
 	var limitedSupportReasonIds []string
+	limitedSupportReasons, err := getLimitedSupportReasons(o.clusterID)
+
+	if len(limitedSupportReasons) == 0 {
+		fmt.Fprintf(os.Stderr, "Cluster does not has any limited support reason ID \n")
+		os.Exit(1)
```

Comment:
**typo:**
```
fmt.Fprintf(os.Stderr, "Cluster does not has any limited support reason ID \n")
```
We could re-phrase to: `Cluster is not in limited support.`

**nitpick:** exit in the main logic by returning the error instead of manually printing the error and calling `os.Exit(1)` here.